app.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+function StoreSalesGenerator(name, minCust, maxCust, avgSale) {
+  this.storeName = name;
+  this.minCust = minCust;
+  this.maxCust = maxCust;
+  this.avgSale = avgSale;
+  this.numCookies = [];
+  this.totalSales = 0;
+  this.hour = 6;
+}
+
+var newText = document.createTextNode('text');
+var tableRow, tableHeader, tableData;
```

Comment:
Consider declaring these variables inside of each function that calls them.  While it is more to write, the variables will only exist in the name space of the function and be removed after the function completes.  As you are not continually accessing these variables this will save memory usage.